docs/style-guide.rst

```diff
@@ -750,3 +750,124 @@ General Recommendations
 =======================
 
 TODO
+
+***********
+NatSpec
+***********
+
+Solidity contracts can have a special form of comments that form the basis of the Ethereum Natural Specification Format. For a usage example please check [here](https://github.com/ethereum/wiki/wiki/Natspec-Example/).
+
+Documentation Example
+=====================
+
+Documentation is inserted above the function following the doxygen notation of either one or multiple lines starting with `///` or a multiline comment starting with `/**` and ending with `*/`.
+
+As an example consider the documentation of the following function::
+
+    /// @notice Send `(valueInmGAV / 1000).fixed(0,3)` GAV from the account of 
```

Comment:
NatSpec was changed years ago to support javascript snippets and not some weird other formatting language.